zeebe/broker/src/main/java/io/camunda/zeebe/broker/system/partitions/PartitionStartupAndTransitionContextImpl.java

```diff
@@ -151,8 +151,10 @@ public PartitionStartupAndTransitionContextImpl(
     this.diskSpaceUsageMonitor = diskSpaceUsageMonitor;
     this.gatewayBrokerTransport = gatewayBrokerTransport;
     this.topologyManager = topologyManager;
-    this.brokerMeterRegistry = new CompositeMeterRegistry().add(brokerMeterRegistry);
-    this.brokerMeterRegistry.config().commonTags(Tags.of("partition", String.valueOf(partitionId)));
+    this.startupMeterRegistry = new CompositeMeterRegistry().add(startupMeterRegistry);
+    this.startupMeterRegistry
+        .config()
+        .commonTags(Tags.of("partition", String.valueOf(partitionId)));
```

Comment:
:wrench: should we use a static field for the partition name as it's shared by both partitionMeterRegistries? This should be the same tag as `PartitionKeyNames`